### PR TITLE
Deprecation warning fix for Kaminari

### PR DESCRIPTION
**Pull request by benbristow**

### CODE REVIEW

**Comment by timoschilling** [comment]
What effect does this change have to the required kaminari version / which is the min version which supports `total_pages`?

**Comment by sublimecoder** [comment]
@timoschilling looks like the changes was introduced in Kaminari 0.14.0  So the earliest supported version should be that one.

https://github.com/amatsuda/kaminari/blob/master/CHANGELOG.rdoc#0140

**Comment by timoschilling** [comment]
Ok, we are already limited to Kaminari 0.15

File: spec/unit/views/components/paginated_collection_spec.rb [hunk 1]
```diff
@@ -212,7 +212,7 @@ def paginated_collection(*args)
 
       describe "set to false" do
         it "should not show the total item counts" do
-          expect(collection).not_to receive(:num_pages)
+          expect(collection).not_to receive(:total_pages)
           expect(collection).not_to receive(:total_pages)
```
Comment (timoschilling) [inline spec/unit/views/components/paginated_collection_spec.rb]: this row is doubled

**Comment by stadia** [comment]
#4427

**Comment by sublimecoder** [comment]
@benbristow Can you fix the line duplication so we can get this merged in to fix the deprecation warnings?

**Pull request merged**